src/runtime/tests/manifest-parser-test.ts

```diff
@@ -57,6 +57,13 @@ describe('manifest parser', () => {
         h1: create 'my-id' #anotherTag @ttl(1h)
         h2: create @ttl ( 30m )`);
   });
+  it('parses recipes with a synthetic join handles', () => {
```

Comment:
Probably worth having a test that it throws when joining over non-handles

Reply:
Added in manifest-test, as the parser itself does not try to understand what the joined identifiers mean.

Reply:
Thanks, perfect.